practice-app/website/api/event.py

```diff
@@ -1,12 +1,12 @@
-from flask import Blueprint, request,  url_for, jsonify, make_response, abort, flash
+from flask import Blueprint, json, request,  url_for, jsonify, make_response, abort, flash
 from ..models import Event
 from .. import db
 import requests
 from flasgger.utils import swag_from
 from sqlalchemy import exc
 events = Blueprint('events', __name__)
 
-API_KEY = '<api key>'
+API_KEY = "AIzaSyB918Ru6ZsU1P3OPc-IaE_lRLgjNM1Suyk"
```

Comment:
I will change the API key, before merging place add a placeholder here.

Reply:
done